framework/src/headers/mac.rs

```diff
@@ -153,3 +169,45 @@ impl MacHeader {
         self.dst.copy_address(&src);
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
Should we use `proptest` in netbricks core? This seems like a good candidate for that.

Reply:
after talking to @clinedome, will get this and a few other proptest things in from @scrollins soon in a separate pr.